lib/shared/drive-constraints.js

```diff
@@ -208,3 +208,115 @@ exports.isDriveValid = (drive, image) => {
 exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE);
 };
+
+/**
+ * @summary Drive/image compatibility status messages.
+ * @public
+ *
+ * @description
+ * Status messages intended to be shown to the user.
+ *
+ * @type {Object}
+ */
+exports.COMPATIBILITY_STATUS_MESSAGES = {
+  OK: '',
+  NOT_RECOMMENDED: 'Not Recommended',
+  TOO_SMALL: 'Too Small For Image',
+  LOCKED: 'Locked',
+  SYSTEM: 'System Drive',
+  CONTAINS_IMAGE: 'Drive Contains Image'
+};
+
+/**
+ * @summary Drive/image compatibility status types.
+ * @public
+ *
+ * @description
+ * Status types classifying what kind of message it is, i.e. ok, error, warning.
+ *
+ * @type {Object}
+ */
+exports.COMPATIBILITY_STATUS_TYPES = {
```

Comment:
Perhaps doing just:
```js
exports.COMPATIBILITY_STATUS_TYPE_OK = 0;
exports.COMPATIBILITY_STATUS_TYPE_ERROR = 1;
exports.COMPATIBILITY_STATUS_TYPE_WARNING = 2;
```
would feel more like Javascript and less like C? \*shrug\*